


I think that exception always existed and is harmless.
Is there anything broken?
Fixed by luco with this changeset:
https://github.com/mozilla/r2d2b2g/commit/c07aae12ddfbea06661e4cc08a63293e1cebfa78
